docs/source/index.rst

```diff
@@ -0,0 +1,20 @@
+`idelib`: a Python Library for accessing enDAQ sensor recordings
+==============================================================
+
+`idelib` is a Python API for accessing `enDAQ <https://endaq.com/>`_'s IDE recordings. The IDE format is an `EBML <https://github.com/ietf-wg-cellar/ebml-specification>`_ encoded file using a custom schemata. This library utilizes our `ebmlite <https://github.com/MideTechnology/ebmlite>`_ to parse the files, and provides classes that make reading data simple.
```

Comment:
Minor: "Schemata" is plural. The line should either be 'using a custom schema' or 'using custom schemata.'

Reply:
good catch; the enDAQ Lab docs should be fixed in the same way

Reply:
addressed in 65a79aa993e7c14739a4064c41538991e9bdd2ed